test/document_stores/test_document_store.py

```diff
@@ -1447,3 +1447,30 @@ def test_normalize_embeddings_diff_shapes():
     VEC_1 = np.array([0.1, 0.2, 0.3], dtype="float32").reshape(1, -1)
     BaseDocumentStore.normalize_embedding(VEC_1)
     assert np.linalg.norm(VEC_1) - 1 < 0.01
+
+
+@pytest.mark.parametrize("document_store_with_docs", ["memory"], indirect=True)
+def test_update_bm25(document_store_with_docs):
```

Comment:
Let's rename to `test_memory_update_bm25`, remove the parametrization and init the docstore into the test. This way we don't need the distinction between `memory` and `memory_bm25`